app/presenters/licences/view-licence-contact-details.presenter.js

```diff
@@ -7,6 +7,11 @@ const { filteredContactDetailsByRole } = require('../crm.presenter.js')
  * @module ViewLicenceContactDetailsPresenter
  */
 
+const ROLES = {
```

Comment:
Just so they don't get confused with the contact roles in the metadata, I'd suggest going a bit more verbose.

```suggestion
const ENTITY_ROLES = {
```